examples/ssh/backup_ftps.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * This driver is designed to verify if backup exist and have a certain size.
+ * Communication protocol is SSH.
+ * This driver create a dynamic monitoring variables 
+ * fileSize: to store file size.
+ * lastodif: to store last modification date.
```

Comment:
What about tested platforms?

How do you know if a device with SSH actually this protocol running at all?